SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularitySlaveAndRackManager.java

```diff
@@ -170,6 +170,7 @@ public SlaveMatchState doesOfferMatch(Protos.Offer offer, SingularityTaskRequest
     switch (slavePlacement) {
       case SEPARATE:
       case SEPARATE_BY_DEPLOY:
+      case SPREAD_ALL_SLAVES:
         if (numOnSlave > 0 || numCleaningOnSlave > 0) {
           LOG.trace("Rejecting SEPARATE task {} from slave {} ({}) due to numOnSlave {} numCleaningOnSlave {}", taskRequest.getRequest().getId(), slaveId, host, numOnSlave, numCleaningOnSlave);
```

Comment:
This log message is not clear when using `SPREAD_ALL_SLAVES` which does not contain the word `SEPARATE` and might lead people to think it's using the wrong strategy.